product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISLoginSuccessAfterMultipleFailuresAlertTestCase.java

```diff
@@ -102,43 +102,61 @@ protected void init() throws Exception {
         } while (getActiveExecutionPlanCount() != activeExecutionPlanCount + 1);
     }
 
-    @Test(groups = "wso2.analytics.is", description = "Publishing sample events for suspicious login alerts")
+    @Test(groups = "wso2.analytics.is", description = "P"
```

Comment:
FIxed with https://github.com/wso2/analytics-is/pull/287/commits/8cec5f64ca1f952edb94a36ae36a3cb878dbac6f